js/interface.js

```diff
@@ -215,21 +215,32 @@ function loadAppStoreData() {
   }
 
   //try to automaticaly login
-  var $loginButton = $('.login-appStore-button');
-  $loginButton.html('Logging in');
-  $loginButton.addClass('disabled');
-  $('#fl-store-appDevPass').addClass('disabled');
-  $('#fl-store-appDevLogin').addClass('disabled');
-  getCredential(organizationID, appStoreSubmission.data['fl-credentials']).then(function (credential) {
-    return appStoreTeamSetup(credential.email, $loginButton);
-  }).catch(function (error) {
-    //we don't need to handle errors for automatic login
-    $loginButton.html('Log in');
-    $('#fl-store-appDevLogin').removeClass('disabled');
-    $('#fl-store-appDevPass').removeClass('disabled');
-    $loginButton.removeClass('disabled');
-    Fliplet.Widget.autosize();
-  });
+  if (appStoreSubmission.data && appStoreSubmission.data['fl-credentials'] && appStoreSubmission.data['fl-credentials'] !== '') {
```

Comment:
`appStoreSubmission.data['fl-credentials'] !== ''` is unnecessary since `appStoreSubmission.data['fl-credentials']` in a conditional check return `false` for an empty string :)